cypress/integration/document_level_monitor_spec.js

```diff
@@ -122,42 +124,59 @@ describe('DocumentLevelMonitor', () => {
       cy.contains(sampleDocumentLevelMonitor.triggers[0].document_level_trigger.name);
 
       // Go back to the Monitors list
-      cy.get('a').contains('Monitors').click();
+      cy.get('a').contains('Monitors').click({ force: true });
 
       // Confirm we can see the created monitor in the list
       cy.contains(SAMPLE_EXTRACTION_QUERY_MONITOR);
     });
 
     it('by visual editor', () => {
       // Select visual editor for method of definition
-      cy.get('[data-test-subj="visualEditorRadioCard"]').click();
+      cy.get('[data-test-subj="visualEditorRadioCard"]').click({ force: true });
 
       // Wait for input to load and then type in the monitor name
       cy.get('input[name="name"]').type(SAMPLE_VISUAL_EDITOR_MONITOR);
 
       // Wait for input to load and then type in the index name
       cy.get('#index').type(`${TESTING_INDEX}{enter}`, { force: true });
 
-      // Enter query name
+      // Enter first query name
       cy.get('[data-test-subj="documentLevelQuery_queryName0"]').type(
         sampleDocumentLevelMonitor.inputs[0].doc_level_input.queries[0].name
       );
 
-      // Enter query field
+      // Enter first query field
       cy.get('[data-test-subj="documentLevelQuery_field0"]').type('region{downarrow}{enter}');
 
-      // Enter query operator
-      cy.get('[data-test-subj="documentLevelQuery_operator0"]').type('is{enter}');
+      // Enter first query operator
+      cy.get('[data-test-subj="documentLevelQuery_operator0"]').select('is');
 
-      // Enter query
+      // Enter first query
       cy.get('[data-test-subj="documentLevelQuery_query0"]').type('us-west-2');
 
-      // Enter query tags
-      cy.get('[data-test-subj="addDocLevelQueryTagButton_query0"]').click().click();
+      // Enter first query tags
+      cy.get('[data-test-subj="addDocLevelQueryTagButton_query0"]').click({ force: true });
       cy.get('[data-test-subj="documentLevelQueryTag_text_field_query0_tag0"]').type(
         sampleDocumentLevelMonitor.inputs[0].doc_level_input.queries[0].tags[0]
       );
 
+      // Add a second query
+      cy.get('[data-test-subj="addDocLevelQueryButton"]').click({ force: true });
+
+      // Enter second query name
+      cy.get('[data-test-subj="documentLevelQuery_queryName1"]').type(
+        sampleDocumentLevelMonitor.inputs[0].doc_level_input.queries[1].name
+      );
+
+      // Enter second query field
+      cy.get('[data-test-subj="documentLevelQuery_field1"]').type('numberField{downarrow}{enter}');
+
+      // Enter second query operator
+      cy.get('[data-test-subj="documentLevelQuery_operator1"]').select('is less than');
+
+      // Enter second query
+      cy.get('[data-test-subj="documentLevelQuery_query1"]').type(1000);
+
```

Comment:
Can we check all the new operators, not just `less than`?

Reply:
Revised the PR to include those test cases.